Addons/Entitas.CodeGeneration.Unity.Editor/Entitas.CodeGeneration.Unity.Editor/Editor/UnityCodeGenerator.cs

```diff
@@ -63,7 +63,7 @@ public static class UnityCodeGenerator {
             }
 
             var assembly = typeof(UnityEditor.Editor).Assembly;
-            var logEntries = assembly.GetType("UnityEditorInternal.LogEntries");
+            var logEntries = assembly.GetType("UnityEditor.LogEntries");
```

Comment:
Hey! Thanks.
Could you please check if there's sth like `UNITY_2017` so we can use this compiler flag? Otherwise this fix will break it for Unity versions < Unity 2017